### ISSUE TITLE
On iOS 11 jailbroken by electra, it pops "Cert Time Needs to Be Updated", Why?

### ISSUE THREAD

**Issue by seeskyline** [lululombard/CertRemainTime#13]
Does it work on iOS 11 electra? It always pops "Cert Time Needs to Be Updated to work with iOS 11". Could you tell me why and how to fix it?

Thanks!

**Comment by lululombard** [comment]
Hi !
Yes it should work, could you upload a screenshot of the shown message ?
Thanks !

EDIT : Just asked BigBoss about the issue, I think it's due to their repo.

**Comment by seeskyline** [comment]
It shows like this below:
![wechatimg408](https://user-images.githubusercontent.com/2876892/35776193-111debb0-09d3-11e8-81b6-87957c946339.jpeg)

**Comment by seeskyline** [comment]
It shows like this below
![wechatimg409](https://user-images.githubusercontent.com/2876892/35776203-3c198770-09d3-11e8-851c-71e2ee502652.jpeg)

**Comment by lululombard** [comment]
Apparently this has to do with the fact that iOS 11 removed the ability to run 32 bits binaries. I’ll look into that later, but I think that this can easily be fixed. I think that this must be set in the Makefile : « ARCHS = armv7 arm64 ».

**Comment by FaZeIlLuMiNaTi** [comment]
To get the app working on iOS 11, the following steps should be followed.

Install the app as you would other apps (extract the .deb and copy the .app folder to /Applications).

Open an SSH or terminal session and run the following commands.

```
wget -O /bootstrap/platform.ent - http://pastebin.com/raw/pnMaaAme
```
OpenSSL is not compiled for 64-bit, meaning we can't use SSL to download - we're downloading this file via HTTP and should check the MD5 hash.
Make sure the output of this command matches `MD5 (/bootstrap/platform.ent) = 02ba0aa7cdaf39423e7077224c6ceed0`.
```
md5 /bootstrap/platform.ent
```
```
jtool -e arch -arch arm64 /Applications/CertRemainTime.app/CertRemainTime
```
```
mv /Applications/CertRemainTime.app/CertRemainTime /Applications/CertRemainTime.app/CertRemainTime.old
```
```
mv /Applications/CertRemainTime.app/CertRemainTime.arch_arm64 /Applications/CertRemainTime.app/CertRemainTime 
```
```
jtool --sign --ent /bootstrap/platform.ent --inplace /Applications/CertRemainTime.app/CertRemainTime
```
```
uicache
```
These steps download an entitlement file, make the application arm64 and sign the application with the correct entitlements.

If anyone follows these steps and has success, let me know.

# This is now irrelevant with the release of Electra + Cydia

**Comment by lululombard** [comment]
Should be fixed with PR #15
